docs/transformations.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Transformations
+---
+
+Transformations are a powerful new Svix feature that allow the modification of certain webhook properties in-flight. When you enable Transformations, your customers can write Javascript code on their endpoints that can change a webhook's HTTP method, target URL, and body payload.
+
+## Enabling Transformations
+
+Transformations can be enabled at the environment-level. When you enable Transformations for an environment, your customers will be able to use Transformations on their endpoints.
+
+To enable Transformations, navigate to the Settings page in the dashboard. Under the Environment Settings section, click General Settings, and then toggle the Enable Transformations switch. Your customers will need to reenter the Consumer App Portal for the change to take effect.
+
+![Environment Settings page with the Enable Transformation toggle](./img/enable-transformations.png)
+
+## Using Transformations
+
+Once enabled for an environment, you customers can begin using Transformations by logging into the Consumer Portal, clicking on an endpoint, clicking into the Advanced tab, and scrolling down to the Transformations card:
+
+![The Transformations card on an endpoint](./img/transformations-card.png)
```

Comment:
Can you retake these screenshots without the beta tag showing? Just so they are evergreen.